src/Apex_Assignment_03/classes/TrainingSessionManager.cls

```diff
@@ -0,0 +1,236 @@
+/**
+ * Description : Created this class for implementing DML Operations and SOQL Queries
+ *
+ * Created By : Yatika Dewaskar
+ * 
+ * Created Date : 11/27/2024
+ * 
+ * Revision Logs : V1 Created
+ * 
+ **/
+public without sharing class TrainingSessionManager {
+    /*
+    * Description : Method to create new record and insert into database 
+    * Created By : Yatika Dewaskar
+    * Created Date : 11/27/2024
+    * Return type : void 
+    * Revision Logs : V1 Created 
+    */
+    public void createTrainingSessionRecords(){
+        try{
+            List<Training_Session__c> trainingSessionList = new List<Training_Session__c>();
+            for(Integer i = 0 ; i<5 ; i++){
+                Training_Session__c tsc = new Training_Session__c(Name = 'Training Session'+i , Location__c = 'Jaipur');
+                trainingSessionList.add(tsc);
+            }
+            insert trainingSessionList;
+        }
+        catch(DmlException e){
+            System.debug('Exception has Occured '+e.getMessage());
+        }
+    }
+    /*
+    * Description : Method to Update Training_Session__c records.
+
+    * Created By : Yatika Dewaskar
+    * Created Date : 11/27/2024
+    * Return type : void 
+    * Revision Logs : V1 Created 
+    */
+    public void updateTrainingSessionRecord(){
+        try{
+            List<Training_Session__c> trainingSessionList = [SELECT Name FROM Training_Session__c WHERE Location__c = 'Jaipur'];
+            Date d = System.Today() + 30;
+            for(Integer i=0; i<10; i++){
+                Training_Session__c tsc = new Training_Session__c(Name = 'TrainingSession' + ++i + 'Updated', Session_Date__c = d);
+                trainingSessionList.add(tsc);
+            }
+            update trainingSessionList;
+        }
+        catch(DmlException e){
+            System.debug('Exception has Occured '+e.getMessage());
+        }
+    }
+    /*
+    * Description : Method to  delete Training_Session__c records. 
+    * Created By : Yatika Dewaskar
+    * Created Date : 11/27/2024
+    * Return type : void 
+    * Revision Logs : V1 Created 
+    */
+    public void deleteTrainingSessionRecord(){
+        try{
+            List<Training_Session__c> trainingSessionList = [SELECT Name, Session_Date__c, 	Location__c FROM Training_Session__c WHERE Id = 'a05NS000007T97XYAS'];
```

Comment:
remove extra space in query and modify it in such a way that it task Id as an argument and you will use that id in Your where clause.